backend/app/src/admins/admins.module.ts

```diff
@@ -6,7 +6,10 @@ import { AnonymousUserModule } from "../anonymous-user/anonyous-user.module";
 
 @Module({
 	imports: [AnonymousUserModule],
-	providers: [AdminsService],
+	providers: [
+		AdminsService,
+		AnonymousUserService
```

Comment:
I accept this modification, but with caution. 
Next PR will check if service is a singleton with instance id